packages/calcite-components/src/components/tabs/tabs.tsx

```diff
@@ -133,6 +153,30 @@ export class Tabs {
    */
   @State() tabs: HTMLCalciteTabElement[] = [];
 
+  mutationObserver = createObserver("mutation", () => this.updateItems());
+
+  private updateItems = (): void => {
+    const { position, scale } = this;
+
+    Array.from(this.el.querySelectorAll("calcite-tab-nav")).forEach(
+      (nav: HTMLCalciteTabNavElement) => {
+        nav.position = position;
+        nav.scale = scale;
+      }
+    );
+
+    Array.from(this.el.querySelectorAll("calcite-tab")).forEach((tab: HTMLCalciteTabElement) => {
```

Comment:
I think you'll need to update queries to ignore nested tabs.

Reply:
Queries target all descendants, so this will be an issue when you have nested tabs with different props each. Not sure how common `position` and `scale` being different between nested tabs would be, so this could be an edge case.

Reply:
Interim fix, I'll think of a way to have the selector do this for us instead.